### PR TITLE
Fixes for Go 11

### PR DESCRIPTION
**Pull request by drequinox**
Fix issues that result in build failing with Go 1.11x

- update package "github.com/rjeczalik/notify"
- add package "golang.org/x/sys/cpu"
- relevant modifications in 'crypto'

### CODE REVIEW

**Review by jbhurat — DISMISSED** [review]

File: cmd/geth/main.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ import (
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/ethereum/go-ethereum/metrics"
 	"github.com/ethereum/go-ethereum/node"
-	"gopkg.in/urfave/cli.v1"
+	cli "gopkg.in/urfave/cli.v1"
```
Comment (jbhurat) [inline cmd/geth/main.go]: Aliasing `cli` is not required

Comment (drequinox) [reply cmd/geth/main.go]: thanks, I agree,  should I raise a new PR for this or we commit this and raise a quick PR to revert cli alias change if required?

alias, of course, does not have any impact and has nothing to do with go 11 changes.  it was auto-added.

Comment (jbhurat) [reply cmd/geth/main.go]: You can commit the change and push the change from the original branch and it will update the PR

Comment (drequinox) [reply cmd/geth/main.go]: many thanks , updated now.

**Review by jbhurat — DISMISSED** [review]

**Comment by jbhurat** [comment]
@drequinox, can you also update `Dockerfile` and travis config to reflect go 1.11

**Comment by drequinox** [comment]
thanks @jbhurat , updated travis and docker files.

**Review by jbhurat — DISMISSED** [review]

**Review by jpmsam — DISMISSED** [review]

**Review by jpmsam — APPROVED** [review]

**Pull request merged**